ios/RNPhotosFramework/PHCollectionService.m

```diff
@@ -138,7 +138,7 @@ + (NSMutableDictionary *)generateAlbumResponseFromCollection:(PHCollection *)col
         PHAssetCollection *phAssetCollection = (PHAssetCollection *)collection;
         PHAssetCollectionType albumType = [phAssetCollection assetCollectionType];
         PHAssetCollectionSubtype subType = [phAssetCollection assetCollectionSubtype];
-        [albumDictionary setObject:[[RCTConvert PHAssetCollectionTypeValuesReversed] objectForKey:@(albumType)] forKey:@"type"];
+        [albumDictionary setObject:ObjectOrNull([[RCTConvert PHAssetCollectionTypeValuesReversed] objectForKey:@(albumType)]) forKey:@"type"];
```

Comment:
What does this code look like in the repo that we forked it from?

Reply:
Source hasn't touched in 4 years. Same as what it was before.